pegasus/sites.v3/hourofcode.com/public/css/action-blocks.scss

```diff
@@ -1,3 +1,5 @@
+@import 'breakpoints';
```

Comment:
why did this need to be imported?

Reply:
I was only using this stylesheet on the how-to pages before this, and that was already importing this: https://github.com/code-dot-org/code-dot-org/blob/3f73c20d8bd11f4cf378c6532036dedf352c9368/pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss#L1